src/main/webapp/WEB-INF/views/PokedexManagement.jsp

```diff
@@ -0,0 +1,152 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1" pageEncoding="ISO-8859-1" %>
+    <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+        <html>
+
+        <head>
+            <title>Add Pokemon Page</title>
+            <style>
```

Comment:
update your charset, html lang, and meta charsets like other pages for consistency.